indico/modules/search/blueprint.py

```diff
@@ -15,5 +15,4 @@
 _bp.add_url_rule('/search/', 'search', RHSearchDisplay)
 
 # Search APIs
-_bp.add_url_rule('/search/api/<any(event,category,contribution,attachment,event_note):type>',
-                 'api_search', RHAPISearch)
+_bp.add_url_rule('/search/api', 'api_search', RHAPISearch)
```

Comment:
I'm not a big fan of this URL

```suggestion
_bp.add_url_rule('/search/api/search', 'api_search', RHAPISearch)
```